impl/events.go

```diff
@@ -36,19 +34,6 @@ func (m *manager) OnDataReceived(chid datatransfer.ChannelID, link ipld.Link, si
 		return err
 	}
 
-	m.reconnectsLk.RLock()
```

Comment:
Remove code that watches for the connection going down and coming up again, this is now handled by the channel monitor